tools/ci_build/github/azure-pipelines/post-merge-jobs.yml

```diff
@@ -1,143 +1,177 @@
+# these parameters are provided to selectively enable build configurations when running manually
+# all build configurations should be enabled by default
+
+parameters:
+- name: EnableMimalloc
+  displayName: Enable Mimalloc build?
+  type: boolean
+  default: true
+
+- name: EnableNoAbseil
+  displayName: Enable no Abseil build?
+  type: boolean
+  default: true
+
+- name: EnableMinimalWithNoExceptions
+  displayName: Enable minimal build with no exceptions?
+  type: boolean
+  default: true
+
+- name: EnableDebugNodeIo
+  displayName: Enable build with debug node I/O?
+  type: boolean
+  default: true
+
+- name: EnableCodeCoverage
+  displayName: Enable code coverage build?
+  type: boolean
+  default: true
+
 jobs:
-- template: templates/win-ci-vs-2019.yml
```

Comment:
I'd suggest viewing this diff with whitespace ignored

Reply:
ah yes, that is much easier to review.